boards/BMS/Src/App/states/App_InitState.c

```diff
@@ -33,14 +38,14 @@ static void InitStateRunOnTick100Hz(struct StateMachine *const state_machine)
     struct BmsWorld *world = App_SharedStateMachine_GetWorld(state_machine);
     struct Clock *   clock = App_BmsWorld_GetClock(world);
 
-    // After entering the init state for 5 seconds, enter the AIR open state
-    if (App_SharedClock_GetCurrentTimeInMilliseconds(clock) -
-            App_SharedClock_GetPreviousTimeInMilliseconds(clock) >=
-        5000U)
+    // if AIR- closes and the TS voltage is below 5V then begin precharge
+    if (Io_Airs_IsAirNegativeClosed() &&
+        Io_VoltageSense_GetTractiveSystemVoltage() < 5)
```

Comment:
how did we decide here on 5V? is this an arbitrarily chosen voltage?

Reply:
Yeah just figured 5V was low enough

Reply:
5V was low enough in my mind and isn't greatly affected by measurement error. Our voltage readings have pretty large error and low voltages

Reply:
okie